src/pages/lineProfile/LineProfile.tsx

```diff
@@ -0,0 +1,132 @@
+import styled from 'styled-components'
+import Grid from '@mui/material/Unstable_Grid2'
+import { NotFound } from '../components/NotFound'
+import { PageContainer } from '../components/PageContainer'
+import { useTranslation } from 'react-i18next'
+import Widget from 'src/shared/Widget'
+import { useLoaderData } from 'react-router-dom'
+import { MapWithLocationsAndPath } from '../components/map-related/MapWithLocationsAndPath'
+import LineProfileHeader from './LineProfileHeader'
+import { LineProfileDetails } from './LineProfileDetails'
+import { Route } from './Route.interface'
+import { DateSelector } from '../components/DateSelector'
+import { SearchContext } from 'src/model/pageState'
+import { useContext, useEffect, useState } from 'react'
+import moment from 'moment'
+import RouteSelector from '../components/RouteSelector'
+import { getRoutesAsync } from 'src/api/gtfsService'
+import { BusRoute } from 'src/model/busRoute'
+import { useSingleLineData } from 'src/hooks/useSingleLineData'
+import { FilterPositionsByStartTimeSelector } from '../components/FilterPositionsByStartTimeSelector'
+import { Spin, Tooltip } from 'antd'
+import './LineProfile.scss'
+
+const LineProfileWrapper = () => (
+  <PageContainer className="line-data-container">
+    <LineProfile />
+  </PageContainer>
+)
+
+const LineProfile = () => {
+  const { t } = useTranslation()
+  const {
+    search: { timestamp },
+    setSearch,
+  } = useContext(SearchContext)
+  const route = useLoaderData() as Route & { message?: string }
+  const [availableRoutes, setAvailableRoutes] = useState<BusRoute[] | null>(null)
+  const [selectedRouteKey, setSelectedRouteKey] = useState<string>('')
+
+  useEffect(() => {
+    window.scrollTo(0, 0)
+  }, [])
```

Comment:
why is it needed?

Reply:
I don't know why, but when you first arrive on this page, you are not at the top of it (i.e. you don't see the title for example) - see picture

![image](https://github.com/hasadna/open-bus-map-search/assets/93577239/ba2bf1f5-25cb-4c63-93f4-de0a3c5a22a4)

Is there a better way to do it?

Reply:
I'm not sure, we can investigate that later